### PR TITLE
Add README.md + README.en.md

### PR DESCRIPTION
**Pull request by raindropdrop**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -7,5 +7,5 @@
     <!-- For formal released project, please use Dynamic Key
         http://docs.agora.io/en/user_guide/Component_and_Others/Dynamic_Key_User_Guide.html -->
     <!-- Tutorial Step 1 -->
-    <string name="agora_app_id"><#YOUR APP ID#></string>
+    <string name="agora_app_id">aab8b8f5a8cd4469a63042fcfafe7063</string>
```
Comment (guohai) [inline app/src/main/res/values/strings.xml]: Can not put the app id to open-sourced samples

File: app/src/main/jniLibs/x86/PLACEHOLDER [hunk 2]
```diff
@@ -1 +0,0 @@
-libagora-rtc-sdk-jni.so
```
Comment (guohai) [inline app/src/main/jniLibs/x86/PLACEHOLDER]: do not remove PLACEHOLDER files or add SDK binaries to open-sourced samples

File: README.en.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+With this sample app, you can:
+
+- Join / leave channel
+- Mute / unmute audio
+- Use capture session to get camera data
+- Switch camera
+- Enable / disable video
+
+A tutorial demo can be found here: [Agora-Video-Source-Android](https://github.com/AgoraIO/Agora-Video-Source-Android)
```
Comment (guohai) [inline README.en.md]: same url with current project?

File: README.en.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+You can find demo for iOS here: [Agora-iOS-Tutorial-Swift-1to1](https://github.com/AgoraIO/Agora-iOS-Tutorial-Swift-1to1)
```
Comment (guohai) [inline README.en.md]: Should it be https://github.com/AgoraIO/Agora-Video-Source-iOS?

File: README.en.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+## Running the App
+First, create a developer account at [Agora.io](https://dashboard.agora.io/signin/), and obtain an App ID. Update "app/src/main/res/values/strings_config.xml" with your App ID.
```
Comment (guohai) [inline README.en.md]: rename to strings.xml.

So far we have different files(strings.xml or strings_config.xml) upon projects.

File: README.en.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+```
+<string name="private_app_id"><#YOUR APP ID#></string>
```
Comment (guohai) [inline README.en.md]: agora_app_id

File: README.md [hunk 7]
```diff
@@ -1,46 +1,45 @@
...
-- `RtcEngine.setClientRole(int role, String permissionKey)`
-- `RtcEngine.joinChannel(String channelKey, String channelName, String optionalInfo, int optionalUid)`
-- `RtcEngine.leaveChannel()`
-- `RtcEngine.setExternalVideoSource(boolean enable, boolean useTexture, boolean pushMode)`
-- `RtcEngine.pushExternalVideoFrame(AgoraVideoFrame frame)`
+你也可以在这里查看 iOS 平台的示例项目：[Agora-iOS-Tutorial-Swift-1to1](https://github.com/AgoraIO/Agora-iOS-Tutorial-Swift-1to1)
 
-## Developer Environment Requirements
+## 运行示例程序
+首先在 [Agora.io 注册](https://dashboard.agora.io/cn/signup/) 注册账号，并创建自己的测试项目，获取到 AppID。将 AppID 填写进 "app/src/main/res/values/strings_config.xml"
```
Comment (guohai) [inline README.md]: check file name with ZH version

File: README.md [hunk 8]
```diff
@@ -1,46 +1,45 @@
...
 
-- Android Studio 2.0 or above
-- Real devices (Nexus 5X or other devices)
-- Note: Android simulator is NOT supported for rendering video, which is a requirement of this app
+```
+<string name="private_app_id"><#YOUR APP ID#></string>
```
Comment (guohai) [inline README.md]: CHECK

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (raindropdrop) [inline app/src/main/res/values/strings.xml]: fixed

File: README.en.md (hunk 4 above)
Comment (raindropdrop) [inline README.en.md]: fixed

File: README.en.md (hunk 5 above)
Comment (raindropdrop) [inline README.en.md]: fixed

File: README.en.md (hunk 6 above)
Comment (raindropdrop) [inline README.en.md]: fixed

File: README.md (hunk 7 above)
Comment (raindropdrop) [inline README.md]: fixed

File: README.md (hunk 8 above)
Comment (raindropdrop) [inline README.md]: fixed

**Comment by raindropdrop** [comment]
Just fixed all problems and pushed new files. Waiting for further review.

**Comment by raindropdrop** [comment]
Folder address just fixed to jniLibs

**Pull request merged**